packages/client/src/lib.js

```diff
@@ -13,6 +13,16 @@
  * ```
  * @module
  */
+
+// @ts-ignore module with no types
+import toIterable from 'stream-to-it'
+// @ts-ignore module with no types
+import { parallelMap } from 'streaming-iterables'
+
+import all from 'it-all'
+import { CarReader } from '@ipld/car'
+import { TreewalkCarSplitter } from 'carbites/treewalk'
```

Comment:
I think these are going to have to be imported from platform.js to do the right thing in the right env.

Reply:
I am not sure I get it. As far as I can tell, platform web is used to get stuff from globalThis, which is not the case here

Reply:
We have to import from skypack (or other) in the browser don't we?

Reply:
humm, don't we ship a bundle of the client ourselves?

@ipld/car and @carbites already ship browser compatible ESM code. If we bundle, minimize and so on, I do not see the reason to use a tool like skypack? But, I am not sure about other potential benefits

Reply:
Yeah ignore me I think I'm just talking rubbish, the js CDNs deal with it for us.